packages/ui/src/components/va-input/hooks/useCleave.ts

```diff
@@ -22,7 +22,7 @@ const DEFAULT_MASK_TOKENS: Record<string, Record<string, unknown>> = {
 }
 
 export const useCleaveProps = {
-  mask: { type: [String, Object] as PropType<string | Record<string, number[]>>, default: '' },
+  mask: { type: [String, Object] as PropType<string | Record<string, number[] | string[]> | boolean>, default: '' },
```

Comment:
```suggestion
  mask: { type: [String, Object] as PropType<string | Record<string, number[] | string[] | boolean>>, default: '' },
```

Typo here. As I understand, you want to handle this case:
```ts
{  creditCard: true }
```

Reply:
I would also appreciate `CleaveOptions & Record<string, any>` type, which will add suggestions in IDE, but still allow passing something not defined in CleaveOptions (in case if our typing is old). What do you think about it?